src/BloomTests/UpdateVersionTableTests.cs

```diff
@@ -25,6 +25,16 @@ public void ThisVersionTooLarge_ReturnsEmptyString()
 		}
 
 		[Test]
+		public void UpgradeTableError_NoCrash()
+		{
+			var t = new UpdateVersionTable();
+			t.TextContentsOfTable = @"0.0.0,3.1.99999, http://first.com/first";
+			t.RunningVersion = Version.Parse("3.2.0");
+			var url = t.LookupURLOfUpdate();
```

Comment:
presumably the steps above put the update lookup table in a state where Error is null, thus verifying that the second test of Error.Status does not unexpectedly happen.

Presumably, url (the result of LookupURLOfUpdate) is not actually a URL, but an UpdateTableLookupResult. And you're not testing an error state of UpgradeTable but its successful operation. But this is not obvious from the name of the variable or the name of the test method.

How about something like lookupResult as the variable name and for the method LookupURLOfUpdate_AllWell_ReportsNoConnectivityError?

Will this test fail if the computer running it is offline? Is it possible/worthwhile to do anything about it if so?